src/collections/programs/communitybridge/index.mdx

```diff
@@ -3,6 +3,7 @@ title: "CommunityBridge"
 link: "communitybridge"
 thumbnail: ./communitybridge.png
 published: true
+program: "CommunityBridge"

```

Comment:
nit: It should be `Community Bridge`

Reply:
@Nikhil-Ladha Should I do it for the title too. Because it was CommunityBridge previously

Reply:
Ohh, my bad. It seems to be like one word, together as `CommunityBridge` only. Do revert this change to `CommunityBridge`.